libvast/src/system/configuration.cpp

```diff
@@ -56,9 +56,11 @@ configuration::configuration() {
     path::current() / "vast.conf",
   };
   auto binary = detail::objectpath();
+#if !VAST_LINUX
```

Comment:
```suggestion
#ifndef VAST_LINUX
```

Reply:
This suggestion is incorrect. Ignore it.

Reply:
Yep, figured the same. We switched to `#cmakedefine01` for a reason.

Reply:
Ah, I missed that transition. Sorry for the noise.